### ISSUE TITLE
`meta` calling forms are broken in the REPL

### ISSUE THREAD

**Issue by scolsen** [carp-lang/Carp#818]
Currently, forms that call `meta` are broken in the REPL:

```
(hidden foo)
(hidden? foo)
`meta` failed, I can’t find `name` at dummy-file:0:0.

Traceback:
(meta name "hidden") at /Users/scottolsen/dev/Carp/core/Macros.carp:43:33.
(list (quote list?) (meta name "hidden")) at /Users/scottolsen/dev/Carp/core/Macros.carp:43:20.
(list (quote not) (list (quote list?) (meta name "hidden"))) at /Users/scottolsen/dev/Carp/core/Macros.carp:43:9.
(eval (list (quote not) (list (quote list?) (meta name "hidd...) at /Users/scottolsen/dev/Carp/core/Macros.carp:43:3.
(hidden? foo) at REPL:1:1.
```

Looks like `meta` isn't being called against the right values?

**Comment by scolsen** [comment]
The fix should be pretty easy--looks like these forms weren't update to account for the new evaluator. Wrapping everything in a `list` instead of calling `meta` directly and calling eval should do the trick.

**Comment by scolsen** [comment]
Fixed by the inimitable @hellerve in #820!
